src/multi_vector_simulator/A1_csv_to_json.py

```diff
@@ -325,10 +333,42 @@ def create_json_from_csv(
                 column_parameters = parameters + extra
                 # check if required parameters are missing
                 for i in set(column_parameters) - set(df_copy.index):
-                    raise MissingParameterError(
-                        f"In file {filename}.csv the parameter {i}"
-                        f" in column {column} is missing."
-                    )
+                    if i == THERM_LOSSES_REL:
```

Comment:
Nice from a short glance (review in January)! I think the automated values can be simplified, then, too.

Reply:
Done with commits: https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/bbf88f91a26ec5eecd6a1755c3430f969a932320, https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/84c67b007391fa762dbbe5121704d6bd5c325708, https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/6653e311fa945af884d049a93c926594a7ef7a1a, https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/7f534b88802b1cab9312373e055b81fcdd308215, https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/39ac947ec61de805a22245f9858134d586f2a6cc, https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/7f691034043fa70baa67d6163fde5e129a2e03f4, https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/6e138b2834226bfbbe2b7d20aa52d5883261613a, https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/300fc406fec62f5300fd864fd4581782b96918ca, https://github.com/rl-institut/multi-vector-simulator/pull/718/commits/60711ab1f7f26ff1779060a0b33914a9cc7a0ef5